src/styles/styles.css

```diff
@@ -0,0 +1,631 @@
+@plugin "@tailwindcss/forms";
+@plugin "tailwind-scrollbar" {
+  nocompatible: true;
+}
+
+@layer components {
+  [type="checkbox"],
+  [type="radio"],
+  [type="text"],
+  [type="search"],
+  [type="number"] {
+    @apply rounded border-borderNormal text-sm;
+
+    &:hover {
+      @apply border-inputSelected;
+    }
+
+    &:focus {
+      @apply outline-none outline-transparent ring-offset-0 ring-0;
+    }
+  }
+
+  [type="checkbox"],
+  [type="radio"] {
+    @apply text-inputSelected cursor-pointer;
+  }
+
+  [type="text"],
+  [type="search"],
+  [type="number"] {
+    @apply text-textDark p-2 bg-input leading-[1.4] w-full;
+
+    &.extra-content-left {
+      @apply pl-8;
+    }
+
+    &.extra-content-right {
+      @apply pr-8;
+    }
+
+    &.large {
+      @apply text-[15px] p-4 rounded-lg;
+
+      &.extra-content-left {
+        @apply pl-11;
+      }
+
+      &.extra-content-right {
+        @apply pr-11;
+      }
+    }
+
+    &.small {
+      @apply text-[13px] py-1;
+
+      &.extra-content-left {
+        @apply pl-6;
+      }
+
+      &.extra-content-right {
+        @apply pr-6;
+      }
+    }
+  }
+
+  [type="radio"] {
+    @apply rounded-full;
+  }
+
+  [type="checkbox"]:checked,
+  [type="checkbox"]:checked:focus {
+    background: none;
+    background-image: url("data:image/svg+xml,%3csvg viewBox='0 0 16 16' fill='%23ffffff' xmlns='http://www.w3.org/2000/svg'%3e%3cpath d='M12.207 4.793a1 1 0 010 1.414l-5 5a1 1 0 01-1.414 0l-2-2a1 1 0 011.414-1.414L6.5 9.086l4.293-4.293a1 1 0 011.414 0z'/%3e%3c/svg%3e");
+    @apply bg-inputSelected;
+  }
+
+  [type="radio"]:checked,
+  [type="radio"]:checked:focus {
+    background: none;
+    @apply bg-white border-[8px] border-inputSelected;
+
+    &:hover {
+      @apply border-inputSelected bg-white;
+    }
+  }
+
+  .landing [type="checkbox"],
+  .landing [type="checkbox"]:checked,
+  .landing [type="checkbox"]:checked:hover,
+  .landing [type="checkbox"]:checked:focus,
+  .landing [type="checkbox"]:indeterminate:hover {
+    @apply border-white bg-transparent;
+  }
+}
+
+body,
+html,
+#__next {
+  height: 100%;
+}
+
+body {
+  @apply text-textNormal;
+  font-family: "InterVariable", "Inter", ui-sans-serif, system-ui, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol",
+    "Noto Color Emoji" !important;
```

Comment:
I'm not thrilled by how many `!important` declarations there are in the CSS overall, but I can accept it as a "lesser evil" for now, particularly if it was out of scope of this PR to resolve this. Let's avoid anything like this at all costs when building the component library components.

Reply:
Dunno why I am doing this here, going to review now. Also made me realise I should move the font declaration into a theme var as per the standard 🤦